### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by evildmp**

### CODE REVIEW

File: aldryn_jobs/admin.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def get_fieldsets(self, request, obj=None):
                 'fields': ['title', 'slug', 'lead_in']
             }),
             (_('Options'), {
-                'fields': ['category', 'is_active', 'can_apply', 'app_config']
+                'fields': ['category', 'is_active', 'can_apply',]
```
Comment (intellectronica) [inline aldryn_jobs/admin.py]: Is this change really related?

File: docs/user/index.rst [hunk 2]
```diff
@@ -12,43 +12,39 @@ Using Aldryn Jobs
 Add a vacancy
```
Comment (mkoistinen) [inline docs/user/index.rst]: Perhaps we should use "job opening" here too?

File: docs/user/index.rst [hunk 3]
```diff
@@ -12,43 +12,39 @@ Using Aldryn Jobs
...
 
-Visit the Jobs page. You should see that the django CMS toolbar now contains a new item, *Jobs*. Select *Add Vacancy...* from this menu.
+Visit the Jobs page. You should see that the django CMS toolbar now contains a new item, *Jobs*. Select *Add Job Opening...* from this menu.
 
 Provide some basic details, such as:
 
-* the ``Lead in`` is a brief summary of the vacancy, that will be used in lists of vacancies, such
-  as on a jobs landing page
-* a ``Category``
+* the ``Short description`` is a brief summary of the vacancy, that will be used in lists of
```
Comment (mkoistinen) [inline docs/user/index.rst]: and here?
